crates/extensions/tedge_file_system_ext/src/lib.rs

```diff
@@ -41,7 +42,8 @@ impl FsWatchMessageBox {
             FsWatchEvent::DirectoryCreated(path) => path,
         };
 
-        self.log_output(&message);
+        log_message_sent("Inotify", &message);
```

Comment:
A bit surprised by this. But that's okay for now as this actor has to be revised in any case.